### ISSUE TITLE
Add option for different refresh interval while driving

### ISSUE THREAD

**Issue by simontb** [simontb/mojio-home-assistant#14]
After #13 is implemented, another nice feature would be an additional setting for a different refresh interval while driving.

Example:
Refresh every 5 minutes and every minute while driving.

I guess whether the car is driving might be determined by looking at whether there's an active trip. At least it's somewhere in the API, because the Mojio app from Telekom Germany shows that the car is driving.

The default value would be the same as for the normal refresh interval (and defaulting to that default of 5 minutes if that's not specified either.

It should be possible to maintain this parameter with and without the regular refresh interval.

**Comment by Bojkas1985** [comment]
This will be an awesome feature! Couldn't it be shortened even more when driving, for example to 5s? I would like to set up automation for it, where when I arrive home the gate opens and the lights turn on.

**Comment by simontb** [comment]
Pulling every 5 seconds is way too aggressive for the backend. I see two ways of how your scenario could be supported:
- If Mojio has a websocket or something similar, use that when a trip is active
- Further increase pull interval only when getting close to the locations defined in home assistant (like home/work)

**Comment by Bojkas1985** [comment]
This is a very reasonable setting, i.e. if there is an active trip, download data from the trip every 1-5 minutes, if it is within a radius of 5-10 km from home, shorten it to an update every 30 seconds and up to a kilometer every 5-10 seconds, that would it didn't have to be as aggressive as you wrote. When will we see the setting option? :)

**Comment by GSzabados** [comment]
The dongle/app updates every minute!

**Comment by themoosman** [comment]
When the car is moving, `status` will change to `moving`.   Additionally, `ignition_state` will be `true` and `parked` will be `false`. You can grab the current Trip (while moving), but there isn't much data in it.  The averages don't appear until the trip is complete.  All the location information is found on the Vehicle.

I'll grab a current moving trip and post the json.
